ethereum/tests/test_solidity.py

```diff
@@ -70,6 +76,26 @@ def test_library_from_code():
     assert contract.test() == 7
 
 
+@pytest.mark.skipif(not SOLIDITY_AVAILABLE, reason='solc compiler not available')
+@pytest.mark.xfail(TRAVIS, reason='solc compiler not available')
```

Comment:
@hackaugusto why fail on travis? we already install `solc` https://github.com/ethereum/pyethereum/blob/develop/.travis.yml#L5-L8